src/number_format.js

```diff
@@ -164,14 +165,14 @@ class NumberFormat extends React.Component {
 
     //remove negation for regex check
     const hasNegation = num[0] === '-';
-    if(hasNegation) num = num.replace('-', '');
+    if (hasNegation) num = num.replace('-', '');
```

Comment:
We will have to write the logicfor removing the parentheses as well.

Reply:
removePrefixAndSuffix will remove the parentheses so getFloatString should never have parentheses, instead the string will be a valid floating point value.